pkgs/development/python-modules/pyqt/5.x.nix

```diff
@@ -90,11 +137,13 @@ buildPythonPackage rec {
     ++ lib.optional withLocation qtlocation
     ++ lib.optional withSerialPort qtserialport
     ++ lib.optional withTools qttools
-  ;
+  );
 
   buildInputs = with libsForQt5; [
     dbus
+  ] ++ lib.optionals (stdenv.buildPlatform == stdenv.hostPlatform) [
```

Comment:
please clean up the conditionals on staging

Reply:
https://github.com/NixOS/nixpkgs/pull/265573